docs/src/design/chiplets/main.md

```diff
@@ -27,7 +27,14 @@ Note that the order of the requests and responses does not matter, as long as th
 
 ### Chiplet bus constraints
 
-The chiplet bus constraints are defined by the [hash](./hasher.md#hash-chiplet-bus-constraints), [bitwise](./bitwise.md#bitwise-chiplet-bus-constraints), and [memory](./memory.md#memory-row-value) chiplets and the components which send lookup requests ([bitwise](../stack/u32_ops.md#u32and), [memory](../stack/io_ops.md#memory-access-operations), and [cryptographic hash operations](../stack/crypto_ops.md) from the stack, and [hash operations](../decoder/main.md#program-block-hashing) for program block hashing from the decoder).
+The chiplet bus constraints are defined by the components that use it to communicate.
+
+Lookup requests are sent to the chiplets bus by the following components:
+
+- The stack sends requests for [bitwise](./stack/u32_ops.md#u32and), [memory](./stack/io_ops.md#memory-access-operations), and [cryptographic hash operations](./stack/crypto_ops.md).
+- The decoder sends requests for [hash operations](./decoder/main.md#program-block-hashing) for program block hashing.
+
+Responses are provided by the [hash](./chiplets/hasher.md#hash-chiplet-bus-constraints), [bitwise](./chiplets/bitwise.md#bitwise-chiplet-bus-constraints), and [memory](./chiplets/memory.md#memory-row-value) chiplets.
```

Comment:
Are the URLs in this paragraph correct? For example, shouldn't `./chiplets/hasher.md#hash-chiplet-bus-constraints` be just `./hasher.md#hash-chiplet-bus-constraints`?

Reply:
Ooh good catch, thank you. I'll double-check all the URLs